fire/SFMainMod.F90

```diff
@@ -113,33 +106,44 @@ end subroutine fire_model
   subroutine UpdateFireWeather(currentSite, bc_in)
     !
     !  DESCRIPTION:
-    !  Updates the site's fire weather index
+    !  Updates the site's fire weather index and calculates effective windspeed based on 
+    !   vegetation characteristics
+    !  Currently we use tree and grass fraction averaged over whole grid (site) to 
+    !  prevent extreme divergence
 
     use FatesConstantsMod, only : tfrz => t_water_freeze_k_1atm
-    use FatesConstantsMod, only : sec_per_day
+    use FatesConstantsMod, only : sec_per_day, sec_per_min
+    use EDTypesMod,        only : CalculateTreeGrassArea
+
+    ! CONSTANTS:
+    real(r8), parameter :: wind_atten_treed = 0.4_r8 ! wind attenuation factor for tree fraction
```

Comment:
I see that these attenuation factors are defined as local constants both in here and also at subroutine `UpdateEffectiveWindSpeed` in SFFireWeatherMod. Perhaps turn these into global parameters, just to ensure they are always consistent across subroutines?

Reply:
I think this was left in by mistake. I am going to delete them here because they are not used anywhere else.